From 99efc1409f44ada7f6c7ca3e849b86ec47103159 Mon Sep 17 00:00:00 2001
From: Etan Cohen <etancohen@google.com>
Date: Fri, 8 Jun 2018 14:48:48 -0700
Subject: [PATCH 06/32] [RTT] Dump native status information into dumpsys

Native status of RTT operation is collected and uploaded to metrics
but was not printed to dumpsys. Added.

Critical to diagnose RTT failures from dumpsys (bug reports).

Bug: 109943669
Test: run RTT then: adb shell dumpsys wifirtt
Change-Id: Id7fff1522c1c44f2ac44c85fd54dd1043dc53bce
---
 service/java/com/android/server/wifi/rtt/RttMetrics.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service/java/com/android/server/wifi/rtt/RttMetrics.java b/service/java/com/android/server/wifi/rtt/RttMetrics.java
index c1bec8f97..ecc1c4812 100644
--- a/service/java/com/android/server/wifi/rtt/RttMetrics.java
+++ b/service/java/com/android/server/wifi/rtt/RttMetrics.java
@@ -116,6 +116,7 @@ public class RttMetrics {
             return "numCalls=" + numCalls + ", numIndividualCalls=" + numIndividualCalls
                     + ", perUidInfo=" + perUidInfo + ", numRequestsHistogram="
                     + numRequestsHistogram + ", requestGapHistogram=" + requestGapHistogram
+                    + ", statusHistogram=" + statusHistogram
                     + ", measuredDistanceHistogram=" + measuredDistanceHistogram;
         }
     }
-- 
2.17.1

